localization/LANG_EN.h

```diff
@@ -137,24 +137,41 @@
 #define MENU_TEXT_LOT "Log of telegrams"
 
 #define STR_24A_TEXT "24h average"
-
-#define STR516_TEXT "Standard values TSP heating circuit 1"
-#define STR536_TEXT "Standard values TSP heating circuit 2 "
-#define STR556_TEXT "Standard values TSP3"
-#define STR576_TEXT "Standard values TSP 4"
+#define STR500_TEXT "Mo"
+#define STR501_TEXT "Tu"
+#define STR502_TEXT "We"
+#define STR503_TEXT "Th"
+#define STR504_TEXT "Fr"
+#define STR505_TEXT "Su"
+#define STR506_TEXT "Sa"
+#define STR516_TEXT "Standard values TSP"
+#define STR632_TEXT "Holiday period 1: first day"
+#define STR633_TEXT "Holiday period 1: last day"
+#define STR634_TEXT "Holiday period 2: first day"
+#define STR635_TEXT "Holiday period 2: last day"
+#define STR636_TEXT "Holiday period 3: first day"
+#define STR637_TEXT "Holiday period 3: last day"
+#define STR638_TEXT "Holiday period 4: first day"
+#define STR639_TEXT "Holiday period 4: last day"
+#define STR640_TEXT "Holiday period 5: first day"
+#define STR641_TEXT "Holiday period 5: last day"
+#define STR642_TEXT "Holiday period 6: first day"
+#define STR643_TEXT "Holiday period 6: last day"
+#define STR644_TEXT "Holiday period 7: first day"
+#define STR645_TEXT "Holiday period 7: last day"
+#define STR646_TEXT "Holiday period 8: first day"
+#define STR647_TEXT "Holiday period 8: last day"
 #define STR648_TEXT "Holiday operating level HC1"
```

Comment:
HC1 needs to be removed here at STR648_TEXT also if the main names/description should be used for either HC1, HC2 and HC3

Reply:
Thanks! Corrected.